testing/bdd/features/validation/facet.feature

```diff
@@ -0,0 +1,77 @@
+@validation @syntax @sequence @xsd
+Feature: Facet
+  Rules on use of ebuttm:facet and ebuttm:documentFacet
+
+
+  # SPEC-CONFORMANCE: R26 R66 R67
+  # skipped because the semantic validation does not check facets yet
+  @skip
+  Scenario: Invalid term identifier
+    Given an xml file <xml_file>
+    When it has facet1 applied to element <parent1>
+    And facet1 has attribute <link1>
+    And facet1 contains string <term1>
+    And it has facet2 applied to element <parent2>
+    And facet2 has attribute <link2>
+    And facet2 contains string <term2>
+    Then document is invalid
+
+    Examples:
+    | xml_file  | parent1 | link1            | term1   | parent2 | link2            | term2   |
+    | facet.xml | body    | http://link1.com | string1 | body    | http://link1.com | string1 |
+    | facet.xml | body    | http://link1.com | string1 | body    | http://link2.com | string1 |
```

Comment:
I added it because of R67 :  	Elements shall NOT contain more than one facet element referring to the same term.

But I may have interpreted it wrong, is the `term` of R67 referring to the `term identifier` composed of the link and the term ?